template-parts/post/author-info.php

```diff
@@ -10,7 +10,7 @@
 <div class="author-description">
 	<h2 class="author-title">
 		<span class="author-heading"><?php _e( 'Published by', 'twentynineteen' ); ?></span>
```

Comment:
This is not properly translatable. It should be:

```suggestion
		<span class="author-heading"><?php echo esc_html( sprintf( __( 'Published by %s', 'twentynineteen' ), get_the_author() ) ); ?></span>
```

Reply:
With a translator note.

Reply:
Thanks for your input. I'll check and apply the suggestion.

Reply:
Yup, make that change and we can go ahead and merge this ;-)